mlos_bench/mlos_bench/environments/local/local_fileshare_env.py

```diff
@@ -69,6 +69,8 @@ def __init__(self,
             "LocalEnv requires a service that supports file upload/download operations"
         self._file_share_service: SupportsFileShareOps = self._service
 
+        # TODO: Replace these with expand_string util functions?
```

Comment:
not sure about `expand_string()` - we can build a `string.Template` once (e.g., in the constructor) and then just do `.safe_substitute()` multiple times. (not that it will boost the performance much, but still :) )

Reply:
For sure.  That was just quick hacking.  I may throw this away as we discussed elsewhere.
I think the rest of the changes from this hacking branch are mostly in at this point.